global.json

```diff
@@ -1,6 +1,6 @@
 {
     "projects": [ "src" ],
     "sdk": {
-        "version": "1.0.0-preview2-003131"
+        "version": "1.0.3"
```

Comment:
@ChrisMaddock the .NET version in the build script is the version that Cake requires, not what we require. This is the version that Cake ships with 1.19.5, I'm not sure if we should be changing it. What do you think? https://github.com/cake-build/cake/blob/v0.19.5/build.ps1